client/app/scripts/components/search.js

```diff
@@ -71,15 +75,36 @@ class Search extends React.Component {
     if (this.state.value && value === '') {
       value = null;
     }
-    this.setState({value});
+    this.setState({ value });
     this.doSearch(inputValue);
   }
 
+  handleKeyUp(ev) {
+    // If the search query is parsable, pin it when ENTER key is hit.
+    if (ev.keyCode === ENTER_KEY_CODE && parseQuery(this.props.searchQuery)) {
+      trackMixpanelEvent('scope.search.query.pin', {
+        query: this.props.searchQuery,
```

Comment:
User may be searching for service/process names which might be sensitive. While it would be interesting to know the type of query too (metrics vs property vs just a string), I don't think we should send the query itself.